docs/conda.rst

```diff
@@ -15,7 +15,8 @@ Activating Conda
 ----------------
 
 Conda Support is the first feature enabled with :doc:`yaml-config`.
-You can enable it by creating a `readthedocs.yml` file in the root of your repository with the contents:
+You can enable it by creating a `readthedocs.yml`, `readthedocs.yaml`, or `.readthedocs.yml` file in the root of your
```

Comment:
I think the link the to the yaml docs are fine here and we can just simplify this by `Read the Docs configuration file` or just `.readthedocs.yml`. What do you think?

Reply:
I agree with Santos. We should mention just one possible option here and give more details below by listing all possible options as you have done.

Reply:
Ok, should we go for `readthedocs.yml` here? Looks like the simplest option.